content/docs/glossary/cost-per-thousand.md

```diff
@@ -0,0 +1,15 @@
+---
+seo:
+  title: Cost per thousand (CPM)
+  description: CPM is used to measure how much 1000 advertisement impressions costs on one webpage
+  keywords: CPM, cost per thousand, web ads, advertisement, ads
+title: Cost per thousand
+weight: 0
+layout: page
+navigation:
+  show: false
+---
+
+Cost per thousand is a term used by marketers to indicate the price of 1,000 advertisement impressions on one webpage. If the owner of a website charges $3.00 CPM, an advertiser must pay $3.00 for every 1,000 impressions of their ad. The "M" in CPM stands for the word "mille," meaning "thousands" in Latin.
+
+Marketers use CPM most commonly for pricing web ads. Marketers often measure how successful a CPM campaign is by its click-through rate.  An advertisement's success cannot be measured by CTR alone because a reader may view the ad, but not click on it and still be influenced by the advertisement. 
```

Comment:
"CPM is the most common pricing model for display advertising."

The CTR stuff feels a little out of place here, to me.